lib/PgSQL_Variables_Utils.cpp

```
@@ -0,0 +1,54 @@
+	for (auto i = 0; i < PGSQL_NAME_LAST_HIGH_WM; i++) {
+		if (pgsql_tracked_variables[i].status == SETTING_VARIABLE) {
+			if (res.empty()) {
+				res += pgsql_tracked_variables[i].set_variable_name;
+			} else {
+				res += string { "|" } + pgsql_tracked_variables[i].set_variable_name;
+			}
+
+			int idx = 0;
+			while (pgsql_tracked_variables[i].alias[idx]) {
+				if (res.empty()) {
+					res += pgsql_tracked_variables[i].alias[idx];
```

Cppcheck (2.19.0)

[warning] If memory allocation fails, then there is a possible null pointer dereference

(nullPointerOutOfMemory)